### PR TITLE
Added wolfSSL-FAQ directory

### PR DESCRIPTION
**Pull request by jackctj117**
Created a wolfSSL-FAQ directory, All information came from [FAQ](https://www.wolfssl.com/docs/frequently-asked-questions-faq/#How_do_I_find_the_correct_CA_certificate_to_load_into_a_client_for_authenticating_a_SSL/TLS_server?) with slight modifications.

### CODE REVIEW

**Review by dgarske — DISMISSED** [review]
PDF looks great, the HTML doesn't have links. @jackctj117 you want to fix that or should we accept it as-is?

**Comment by JacobBarthelmeh** [comment]
Assigned to @jackctj117 while resolving links.

**Comment by jackctj117** [comment]
Fixed links.

**Review by dgarske — APPROVED** [review]

File: wolfSSL-FAQ/header.txt [hunk 1]
```diff
@@ -0,0 +1,21 @@
+% wolfSSL Tuning Guide ![](logo.png)
```
Comment (JacobBarthelmeh) [inline wolfSSL-FAQ/header.txt:1]: Should this be wolfSSL FAQ?

**Review by JacobBarthelmeh — CHANGES_REQUESTED** [review]

File: wolfSSL-FAQ/header.txt (hunk 1 above)
Comment (jackctj117) [reply wolfSSL-FAQ/header.txt:1]: Whoops! I'll change that.

**Review by JacobBarthelmeh — APPROVED** [review]

**Pull request merged**